web/packages/teleport/src/services/history/history.ts

```diff
@@ -53,24 +53,39 @@ const history = {
     window.location.reload();
   },
 
-  goToLogin(rememberLocation = false) {
-    let url = cfg.routes.login;
+  goToLogin({ rememberLocation = false, withAccessChangedMessage = false }) {
```

Comment:
you can do this as a default value, then we dont have to always define `false false` everywhere

```suggestion
  goToLogin({ rememberLocation = false, withAccessChangedMessage = false } = {}) {
```